src/java.base/unix/classes/sun/nio/ch/UnixAsynchronousSocketChannelImpl.java

```diff
@@ -741,10 +741,12 @@ <V extends Number,A> Future<V> implWrite(boolean isGatheringWrite,
                 return result;
             }
         } catch (Throwable x) {
+            System.out.println("Caught " + x);
```

Comment:
Leftover debug output?

Reply:
Yes, thanks for catching.

Reply:
Fixed in ff404d4b52c5ecf4f00743be1d4d4a266483460c.